style.css

```diff
@@ -1,4 +1,74 @@
 body {
-background-color: #232323
+  color: #ffffff;
+  font-family: 'Ubuntu', sans-serif;
+  font-size: 15px;
+  width: auto;
+  background: url(nieckovsky_image_main.jpg);
+  background-color: #111415;
+  background-size: 100%;
+  background-position: center;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+}
+
+header {
+  width: 100%;
+  height: 50px;
+}
+
+nav {
+  display: inline-block;
```

Comment:
You don't need `display: inline-block` here. `display: inline-block` is used when you would like to have two elements that normally do not fit next to each other display well, inline. `nav` in our markup has no siblings, so there's nothing to inline to.

Reply:
Remnant of my experiments and playing around with that property when trying to get a grasp of it's actual behavior - it seemed not to disrupt anything, so I omitted this one when tidying up. Thanks for the explanation.